paasta_tools/api/api_docs/swagger.json

```diff
@@ -880,6 +884,31 @@
     "InstanceTask": {
       "type": "object"
     },
+    "InstanceStatusAdhoc": {
+      "type": "array",
+      "description": "List of runs associated with job",
+      "items": {
+        "$ref": "#/definitions/AdhocLaunchHistory"
+      }
+    },
+    "AdhocLaunchHistory": {
+        "type": "object",
+        "description": "A single run",
+        "properties": {
+           "launch time": {
+              "type": "string",
```

Comment:
Just a small preference here, but can you use underscores `launch_time`, etc to be consistent with the variables names any everything?